tools/dbus_exporter.py

```diff
@@ -10,7 +10,10 @@ def extract_interfaces_xml(output_dir, executable):
     # If proc is not mounted, set LD_LIBRARY_PATH so that shared/core libs can be found
     env = os.environ.copy()
     if not os.path.exists('/proc/self'):
-        env["LD_LIBRARY_PATH"] = f"src/shared:src/core:{env['LD_LIBRARY_PATH']}"
+        if 'LD_LIBRARY_PATH' not in env:
+            env["LD_LIBRARY_PATH"] = "src/shared:src/core"
+        else:
+            env["LD_LIBRARY_PATH"] = f"src/shared:src/core:{env['LD_LIBRARY_PATH']}"
```

Comment:
This should be moved to the earlier commit. Please make the previous commit perfect.

Reply:
fixup in wrong commit, fixed